<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyStore</class>
 <widget class="QWidget" name="MyStore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="DishesDetail">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>美食1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>美食2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>美食3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>美食4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>美食5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>美食6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>美食7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>美食8</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>美食名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>介绍</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="UpdateButton">
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AddFoodToMenuButton">
      <property name="text">
       <string>增加美食</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ReadTurnoverButton">
      <property name="text">
       <string>查看营业额</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ReturnButton">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Delete_1">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Delete_2">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Delete_3">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Delete_4">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Delete_5">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Delete_6">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Delete_7">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Delete_8">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>341</width>
     <height>114</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>简介</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="StoreNameEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>店铺名</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>好评数</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="IntroductionEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="HighOpitionEdit"/>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="EmployeeEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="AddEmployeeButton">
        <property name="text">
         <string>确认添加</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>员工名</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
